templates/confidential-computing/index.html

```diff
@@ -15,35 +19,51 @@
 {% block content %}
 
   <div class="is-paper">
-    <section class="p-strip is-shallow u-no-padding--bottom">
-      <div class="row">
-        <div class="col-9 col-start-large-4">
-          <div class="p-section--shallow">
-            <h1 class="u-no-margin--bottom">Ubuntu for confidential computing</h1>
-            <p class="p-heading--2">
-              Protect your data in use,
-              <br class="u-hide--small u-hide--medium" />
-              across public and private clouds.
-            </p>
-          </div>
-          <hr class="p-rule" />
-          <a class="js-invoke-modal p-button--positive" href="/contact-us">Contact us to learn more</a>
+    {% call(slot) vf_hero(
+        title_text='Ubuntu <br/>for confidential compute',
+        subtitle_text='Protect your data in use,<br/> across public and private clouds.',
+        layout='50/50'
+      ) -%}
+      {%- if slot == 'cta' -%}
+        <a href="/contact-us" class="js-invoke-modal">Contact us to learn more ›</a>
+      {%- endif -%}
+      {%- if slot == 'image' -%}
+        <div class="p-image-container--3-2 is-cover">
+          {{ image(url="https://assets.ubuntu.com/v1/cf1e2ddb-datacenter-wide-crop.jpeg",
```

Comment:
@josetapadas these are all still missing the `attrs={"class": "p-image-container__image"}` mentioned [here](https://github.com/canonical/ubuntu.com/pull/15026/files#r2059835786)

Reply:
Oh thanks, I've missed this one -.-'